### ISSUE TITLE
[Bug]: Suggestion is not a function in jest test

### ISSUE THREAD

**Issue by iamswain25** [ueberdosis/tiptap#5360]
### Affected Packages

extension-mention

### Version(s)

2.5.0

### Bug Description

running jest test produces error with message
```
Suggestion is not a function
```

### Browser Used

Other

### Expected Behavior

/@tiptap/extension-mention/dist/index.cjs in v2.4.0 is
```
    addProseMirrorPlugins() {
        return [
            Suggestion__default["default"]({
                editor: this.editor,
                ...this.options.suggestion,
            }),
        ];
    },
```
but in 2.5.3 is
```
 addProseMirrorPlugins() {
        return [
            Suggestion({
                editor: this.editor,
                ...this.options.suggestion,
            }),
        ];
    },
```

### Dependency Updates

- [X] Yes, I've updated all my dependencies.

**Comment by nperez0111** [comment]
cjs vs esm is always going to be weird. But it looks like you figured out a solution though?

**Comment by nperez0111** [comment]
The exports look like this:

```js

exports.Suggestion = Suggestion;
exports.SuggestionPluginKey = SuggestionPluginKey;
exports.default = Suggestion;
exports.findSuggestionMatch = findSuggestionMatch;
```

**Comment by iamswain25** [comment]
> cjs vs esm is always going to be weird. But it looks like you figured out a solution though?

not yet, there must be difference in rollup config.

**Comment by nperez0111** [comment]
I think I've resolved this with version 2.5.4
